src/app/features/openletters/index.js

```
@@ -0,0 +1,280 @@
+      .addCase(getLetterCategories.fulfilled, (state, action) => {
+        state.isFetching = false;
+        state.categories = action?.payload;
+      })
+      .addCase(getLetterCategories.rejected, (state, action) => {
+        state.isFetching = false;
+        state.error = action?.payload;
+      })
+      .addCase(getLetterByCategory.pending, (state) => {
+        state.isLetterFetching = true;
+      })
+      .addCase(getLetterByCategory.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)